frontend/tests/controllers/ProblemListTest.php

```diff
@@ -229,6 +229,18 @@ public function testProblemFinderSearch() {
 
         \OmegaUp\Test\Utils::runAggregateFeedback();
 
+        // Filter 0:
+        // - Containing tag-0
```

Comment:
```suggestion
        // - Containing test-tag-0
```

así concuerda con L239

Reply:
Listo